android/src/main/kotlin/clean/news/ui/item/share/ItemShareIntent.kt

```diff
@@ -0,0 +1,48 @@
+package clean.news.ui.item.share
+
+import android.content.Intent
+import clean.news.core.entity.Item
+import clean.news.flow.WithComponent
+import clean.news.flow.WithIntent
+import clean.news.presentation.navigation.NavigationFactory.ItemShareKey
+import clean.news.ui.item.detail.ItemDetailScreen
+import clean.news.ui.item.detail.ItemDetailScreen.ItemDetailComponent
+import dagger.Module
+import dagger.Provides
+import dagger.Subcomponent
+import flow.ClassKey
+import flow.TreeKey
+import nz.bradcampbell.paperparcel.PaperParcel
+import nz.bradcampbell.paperparcel.PaperParcelable
+
+@PaperParcel
+class ItemShareIntent(val item: Item) : ClassKey(),
```

Comment:
`Intent` is not platform agnostic. I'd come up with some more generic name, e.g. `ItemShareScreen`.